inc/dm_easy_mesh.h

```diff
@@ -84,237 +84,2450 @@ class dm_easy_mesh_t {
+	 * @param[in] num The number of AP MLDs to set.
+	 *
+	 * @note Ensure that the `dm` pointer is valid and points to a properly initialized EasyMesh configuration object.
+	 */
+	static void set_num_ap_mld(void *dm, unsigned int num) { (static_cast<dm_easy_mesh_t *>(dm))->set_num_ap_mld(num); }
+    
+	/**!
+	 * @brief Retrieves the access point MLD (Multi-Link Device) at the specified index.
+	 *
+	 * This function returns a pointer to the access point MLD structure located at the given index.
+	 *
+	 * @param[in] index The index of the access point MLD to retrieve.
+	 *
+	 * @returns A pointer to the `dm_ap_mld_t` structure at the specified index.
+	 *
+	 * @note Ensure that the index is within the valid range to avoid undefined behavior.
+	 */
+	dm_ap_mld_t *get_ap_mld(unsigned int index) { return &m_ap_mld[index]; }
+    
+	/**!
+	 * @brief Retrieves a reference to the AP MLD at the specified index.
+	 *
+	 * This function returns a reference to the AP MLD (Access Point Multi-Link Device) object
+	 * located at the given index within the internal storage.
+	 *
+	 * @param[in] index The index of the AP MLD to retrieve.
+	 *
+	 * @returns A reference to the AP MLD object at the specified index.
+	 *
+	 * @note Ensure that the index is within the valid range of stored AP MLDs to avoid undefined behavior.
+	 */
+	dm_ap_mld_t& get_ap_mld_by_ref(unsigned int index) { return m_ap_mld[index]; }
+
+    
+	/**!
+	 * @brief Retrieves the number of BSTA MLD.
+	 *
+	 * This function returns the current number of BSTA MLD.
+	 *
+	 * @returns The number of BSTA MLD.
+	 */
+	unsigned int get_num_bsta_mld() { return m_num_bsta_mld; }
+    
+	/**!
+	 * @brief Retrieves the number of BSTA MLD.
+	 *
+	 * This function returns the number of BSTA MLD from the given dm_easy_mesh_t object.
+	 *
+	 * @param[in] dm Pointer to the dm_easy_mesh_t object.
+	 *
+	 * @returns The number of BSTA MLD.
+	 */
+	static unsigned int get_num_bsta_mld(void *dm) { return (static_cast<dm_easy_mesh_t *>(dm))->get_num_bsta_mld(); }
+
+    
+	/**!
+	 * @brief Retrieves the number of associated station MLDs.
+	 *
+	 * This function returns the current count of associated station MLDs.
+	 *
+	 * @returns The number of associated station MLDs.
+	 */
+	unsigned int get_num_assoc_sta_mld() { return m_num_assoc_sta_mld; }
+    
+	/**!
+	 * @brief Retrieves the number of associated stations in the mesh network.
+	 *
+	 * This function returns the number of associated stations in the mesh network
+	 * by accessing the relevant data from the dm_easy_mesh_t structure.
+	 *
+	 * @param[in] dm Pointer to the dm_easy_mesh_t structure.
+	 *
+	 * @returns The number of associated stations in the mesh network.
+	 */
+	static unsigned int get_num_assoc_sta_mld(void *dm) { return (static_cast<dm_easy_mesh_t *>(dm))->get_num_assoc_sta_mld(); }
+
+    
+	/**!
+	 * @brief Retrieves the DPP object.
+	 *
+	 * @returns A pointer to the dm_dpp_t object.
+	 */
+	dm_dpp_t *get_dpp() { return &m_dpp; }
```

Comment:
For clarity with other DPP objects in the codebase.
```suggestion
	/**!
	 * @brief Retrieves the Data Model DPP object.
	 *
	 * @returns A pointer to the dm_dpp_t object.
	 */
	dm_dpp_t *get_dpp() { return &m_dpp; }
```

Reply:
Have committed this change